apps/web/public/draco/draco_wasm_wrapper.js

```diff
@@ -0,0 +1,2426 @@
+var $jscomp = $jscomp || {};
```

Comment:
what makes this necessary? feels highly abnormal

Reply:
we don't want to load draco JS/WASM from CDN, so we have to set it up locally.

Reply:
thanks google